src/System.Diagnostics.Debug/ref/System.Diagnostics.Debug.cs

```diff
@@ -8,105 +8,461 @@
 
 namespace System.Diagnostics
 {
+    /// <summary>
+    /// Provides a set of methods and properties that help debug your code. This class cannot be inherited.
```

Comment:
Not sure if we say in all static/sealed claases, but in most of them, we do. If you look at the namespaces page like [System namespace](https://msdn.microsoft.com/en-us/library/system(v=vs.110).aspx), you'll see the classes that have that.